src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
 public class WorkWithFile {
+    private static final String BUY_WORD = "buy";
+    private static final String SUPPLY_WORD = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToTheFile(toFileName, createReport(readDataFromFile(fromFileName)));
+    }
+
+    private String[] readDataFromFile(String nameOfFile) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(nameOfFile))) {
+            String temporary;
+            while ((temporary = bufferedReader.readLine()) != null) {
+                stringBuilder.append(temporary).append(" ");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Have some problems to read the file: ", e);
+        }
+        return stringBuilder.toString().split(" ");
+    }
+
+    private String createReport(String[] data) {
+        int supplyCounter = 0;
+        int buyCounter = 0;
+        String[] temporaryArray;
+        for (String partOfData : data) {
+            temporaryArray = partOfData.split(",");
```

Comment:
better make constant for separator and use it in other places

Reply:
don't fixed